config/forest.php

```diff
@@ -4,6 +4,7 @@
     'models_directory' => env('MODEL_DIRECTORY', 'app/Models/'),
     'models_namespace' => env('MODEL_NAMESPACE', 'App\Models\\'),
     'json_file_path'   => env('JSON_FILE_PATH', '.forestadmin-schema.json'),
+    'send_apimap_auto' => env('FOREST_SEND_APIMAP_AUTO', false),
```

Comment:
Variable names should be with complete words ideally, `AUTO` -> `AUTOMATIC`
(See this example: https://github.com/ForestAdmin/forest-rails/blob/c040b7351177757bb6d43a9111bcd175a372c219/lib/forest_liana/engine.rb#L92)

Reply:
👍